projects/cdk/schematics/ng-update/v4/steps/constants/attrs-to-replace.ts

```diff
@@ -47,14 +47,14 @@ export const ATTRS_TO_REPLACE: ReplacementAttribute[] = [
     {
         from: {
             attrName: 'item',
-            withTagNames: ['tui-radio'],
+            withTagNames: ['tui-radio', 'tui-radio-labeled', 'tui-radio-block'],
         },
         to: {attrName: 'value'},
     },
     {
         from: {
             attrName: '[item]',
-            withTagNames: ['tui-radio'],
+            withTagNames: ['tui-radio', 'tui-radio-labeled', 'tui-radio-block'],
```

Comment:
it seems that migration is expected to be more complex :(
```
<tui-radio-block size="s" [item]="item" [hideRadio]="true" [formControl]="control">
                    {{ name }}
</tui-radio-block>
```
to
```
<label tuiBlock appearance="" size="s">
<input tuiRadio type="radio" [value]="item"  [formControl]="control"/>
    {{ name }}
</label>
```

Reply:
yep, all other migration for radio (+ unit tests) will be later in separate PR.

It is just small refactor

Reply:
It will be later:
https://github.com/taiga-family/taiga-ui/pull/8212

This migration is too complex – i need some intermediate steps to finish it)